crates/red_knot_module_resolver/src/typeshed/versions.rs

```diff
@@ -92,27 +93,54 @@ impl TypeshedVersions {
         self.0.is_empty()
     }
 
-    pub fn contains_module(&self, module_name: &ModuleName) -> bool {
-        self.0.contains_key(module_name)
+    fn get_exact(&self, module_name: &ModuleName) -> Option<&PyVersionRange> {
+        self.0.get(module_name)
+    }
```

Comment:
Nit: Just `exact`. `get` prefixes are uncommon in the Rust ecosystem
```suggestion
    fn exact(&self, module_name: &ModuleName) -> Option<&PyVersionRange> {
        self.0.get(module_name)
    }
```